jb/fftw/time_delay_estimator.hpp

```diff
@@ -54,8 +43,21 @@ class time_delay_estimator {
       iplan;
   //@}
 
-  /// Constructor
-  time_delay_estimator(timeseries_type const& a, timeseries_type const& b)
+  /**
+   * Constructs a time delay estimator using @param a and @param b as prototypes
+   * for the arguments.
+   *
+   * The optimal algorithm to compute the  FFTs used in the cross correlation
+   * depends on the size of the input parameters and their memory alignment.
+   *
+   * The FFTW library casts the constness of their arguments to change their
```

Comment:
The fact that we are passing const parameters in jb::fftw::plan is a bug, they should have been non-cost if the underlying library was going to change them.  Consider:

```
  * The FFTW library modifies the arguments to compute the optimal execution plan, do not
  * assume the values are unmodified.
```